### ISSUE TITLE
Support for user groups?

### ISSUE THREAD

**Issue by tony-johnson** [Ullink/simple-slack-api#122]
Hi, there does not appear to be any support for user groups:

https://api.slack.com/methods/usergroups.list

in the simple-slack-api. Unless I missed it somewhere? If not has anybody added support for this?

**Comment by bcorne** [comment]
It is not yet supported but please note that bot users don't have access to group member list (as described here : https://api.slack.com/methods/usergroups.users.list ) . So you'll have to use a true user token to use effectivety benefits from user groups

**Comment by tony-johnson** [comment]
Yes, thanks, I just discovered the limitations on bots access to user groups. Has there been any discussion anywhere on why bots can not access user groups -- it seems like an arbitrary and illogical limitation?

**Comment by bcorne** [comment]
That's up to Slack but you can still use a standad account token to be able to use user groups features
